dcrjson/dcrwalletextcmds.go

```diff
@@ -569,6 +569,22 @@ func NewSetTicketVoteBitsCmd(txHash string, voteBits uint16, voteBitsExt *string
 	}
 }
 
+// SetTicketsVoteBitsCmd is a type handling custom marshaling and
+// unmarshaling of setticketsvotebits JSON RPC commands.
+type SetTicketsVoteBitsCmd struct {
+	TxHashes      string
+	VoteBitsBytes string
```

Comment:
No, the save on terrible JSON encoding overhead in Go it's easiest to just encode them as a flat string and then convert it into slices. Ideally, getticketsvotebits should someday upgrade to this, or we should just ditch the JSON RPC.